core/controllers/editor_test.py

```diff
@@ -554,8 +554,13 @@ def test_exploration_download_handler_for_default_exploration(self):
 
         # Check download to JSON.
         exploration.update_objective('Test JSON download')
-        exp_services._save_exploration(  # pylint: disable=protected-access
-            owner_id, exploration, '', [])
+        exp_services.update_exploration(
+            owner_id, exploration.id, [
+                exp_domain.ExplorationChange({
+                    'cmd': exp_domain.CMD_EDIT_EXPLORATION_PROPERTY,
+                    'property_name': 'objective',
+                    'new_value': 'the objective',
```

Comment:
this needs to be 'Test JSON download' right?

Reply:
Yes!

Reply:
is this correct?